pkg/config/groups.go

```diff
@@ -62,8 +62,8 @@ type rpcConfiguration struct {
 	Network string
 	Address string
 	Enabled bool
-	User    string
```

Comment:
If RPC is going to be used over HTTPS (not only UDS), we'll need to secure it with Auth procedure at some point.

Reply:
Will create an issue specific to that